src/main/pages/portable-workspaces/creating-factories.adoc

```diff
@@ -7,66 +7,72 @@ permalink: creating-factories.html
+A factory created from the dashboard is persisted on {{ site.product_mini_name }} and retained when upgrading to a newer version.
 
-A Factory created from the dashboard will be persisted onto {{ site.product_mini_name }} and kept when upgrading to a newer version.
+To create a factory in the IDE:
+
+. In the IDE, click *Workspace* > *Create Factory*.
+
+Sample factory: https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j[https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j]
 
 [id="create-a-factory-based-on-a-repo"]
-== Create a Factory based on a Repo
+== Creating a factory based on a repository
+
+URL factories work with GitHub and GitLab repositories. By using URL factories, the project referenced by the URL is automatically imported.
 
-[cols="1,4"]
-|===
-|Action | Specify the repo URL. In this case the configuration must be stored inside the repository.
-|Sample Factories | http://che.openshift.io/f?url=https://github.com/eclipse/che +
- http://che.openshift.io/f?url=https://github.com/eclipse/che/tree/language-server +
- http://che.openshift.io/f?url=https://gitlab.com/benoitf/simple-project
-|===
+To create a factory based on a repository:
 
+. Specify the repository URL. Ensure that you store the configuration in the repository.
 
-URL Factories work with GitHub and GitLab repositories. By using URL Factories, the project referenced by the URL is automatically imported.
+Sample factories:
 
-The Factory URL can include a branch or a subfolder. Here is an example of optional parameters:
+http://che.openshift.io/f?url=https://github.com/eclipse/che +
+http://che.openshift.io/f?url=https://github.com/eclipse/che/tree/language-server +
+http://che.openshift.io/f?url=https://gitlab.com/benoitf/simple-project
 
-* `?url=https://github.com/eclipse/che` che will be imported with master branch
-* `?url=https://github.com/eclipse/che/tree/5.0.0` che is imported by using 5.0.0 branch
-* `?url=https://github.com/eclipse/che/tree/5.0.0/dashboard` subfolder dashboard is imported by using 5.0.0 branch
+The factory URL can include a branch or a subdirectory. Following are examples of optional parameters:
+
+`?url=https://github.com/eclipse/che`::
+	Che is imported with the `master` branch.
+
+`?url=https://github.com/eclipse/che/tree/5.0.0`:: 
+	Che is imported by using the `5.0.0` branch.
+
+`?url=https://github.com/eclipse/che/tree/5.0.0/dashboard`:: 
+	Subdirectory `dashboard` is imported by using the `5.0.0` branch.
 
 [id="customizing-url-factories"]
-=== Customizing URL Factories
+=== Customizing URL factories
 
-There are 2 ways of customizing the runtime and configuration.
+Following are two ways to customize the runtime and configuration:
 
-*Customizing only the runtime*
+* Customizing only the runtime
 
-*Dockerfile works on Docker infra only!*
+* Dockerfile works on Docker infrastructure only
```

Comment:
If this is the 2nd way to customize the runtime, then where is it described below? I only see "Customizing the project and runtime".